shims/spark310/src/main/scala/com/nvidia/spark/rapids/shims/spark310/Spark310Shims.scala

```diff
@@ -133,7 +133,7 @@ class Spark310Shims extends Spark301Shims {
   }
 
   override def getExecs: Map[Class[_ <: SparkPlan], ExecRule[_ <: SparkPlan]] = {
-    Seq(
+    super.getExecs ++ Seq(
```

Comment:
Fix the faliures about window tests for the regular 310 spark.

Reply:
@tgravescs do you have any issues with this?  If we get breaking changes between 301 and 310 we might need to change this, but I wanted to check with you that you don't see any other issues before merging.

Reply:
it seems fine until something changes that breaks it.